cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -159,10 +160,23 @@ Qcow2 images location:
 		if opt.ImageDist != "coreos" && opt.OSPassword == "" && !opt.OSPasswordSkip {
 			var err error
 			opt.OSPassword, err = GeneratePassword(12)
-			klog.Infof("Autogenerated OS root password is: %s", opt.OSPassword)
 			if err != nil {
 				return err
 			}
+			passwordfile, err := os.Create("password.txt")
```

Comment:
can you please close the file in the defer for safer side?

Reply:
Done.